README.md

```diff
@@ -79,6 +79,16 @@ let line = data |> Js.Json.parseExn
 ```
 
 #### Encode and decode Base64
+
+To encode and decode Base64, you can bind to Javascript functions `btoa` and `atob`, respectively:
+
+```ml
+external  btoa : string -> string = "window.btoa" [@@bs.val]
+external  atob : string -> string = "window.atob" [@@bs.val]
+
+let _ = print_endline @@ atob @@ btoa "Hello World!"
```

Comment:
I think this would be easier to understand if you use the pipe operator (`|>`) instead. `@@` is a bit obscure and much less intuitive.